dk-installer.py

```diff
@@ -1051,19 +1077,19 @@ def execute(self, args):
 
         try:
             with self.start_cmd(
-                "minikube",
-                "kubectl",
-                "--profile",
-                args.profile,
-                "--",
-                "--namespace",
-                args.namespace,
-                "--address",
-                "0.0.0.0",
-                "port-forward",
-                "service/observability-ui",
-                f"{args.port}:http",
-                raise_on_non_zero=False,
+                    "minikube",
+                    "kubectl",
+                    "--profile",
+                    args.profile,
+                    "--",
+                    "--namespace",
+                    args.namespace,
+                    "--address",
+                    "0.0.0.0",
+                    "port-forward",
+                    "service/observability-ui",
+                    f"{args.port}:http",
+                    raise_on_non_zero=False,
```

Comment:
Is this change intentional? The indentation seems off (8 spaces).

I actually see this in a few other places.